documentation/docs/welcome.md

```diff
@@ -0,0 +1,47 @@
+---
+description: INX-Participation is an extension for nodes to enable on-tangle voting.
+image: /img/Banner/banner_hornet.png
+keywords:
+- IOTA Node
+- Hornet Node
+- INX
+- Participation
+- IOTA
+- Shimmer
+- Node Software
+- Welcome
+- explanation
+---
+
+# Welcome to INX-Participation
+
+INX-Participation is an extension for nodes to enable on-tangle voting. 
+The extensions maintains its own database separate from that of the node and provides means to track events and votes.
```

Comment:
```suggestion
The extensions maintains its own database separate from that of the node and provides means to track events and votes.
```

The database part sounds like a detail of implementation. I would move it to the very end of the paragraph or remove it altogether if it is not actually important to know.